languages/csharp/exercises/concept/interfaces/.docs/after.md

```diff
@@ -0,0 +1,170 @@
+`interfaces` are the primary means of [decoupling][wiki-loose-coupling] the uses of a class from its implementation. This decoupling provides flexibility for maintenance of the implementation and helps support type safe generic behavior.
+
+The syntax of an interface is similar to that of a class or struct except that methods and properties appear as the signature only and no body is provided.
+
+```csharp
+public interface ILanguage
+{
+    string Speak();
+}
+
+public interface IItalianLanguage : ILanguage
+{
+    string Speak();
```

Comment:
Did you want to show that interfaces can have the same name? If not, you could consider naming this `SpeakItalian` to show that interfaces can add new methods to existing interfaces.

Reply:
Done